benchmark/src/lib-hexl-benchmark.cpp

```diff
@@ -39,23 +39,13 @@
 #define _USE_MATH_DEFINES
```

Comment:
This file is moving to the openfhe-hexl repo, so we'll need to merge over to there.